cli/cli_integration_test.go

```diff
@@ -0,0 +1,255 @@
+package cli_test
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/featurebasedb/featurebase/v3/cli"
+	"github.com/featurebasedb/featurebase/v3/dax/server/test"
+	"github.com/featurebasedb/featurebase/v3/logger"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCLIIntegration(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping integration test")
+	}
+
+	ctx := context.Background()
+
+	t.Run("Stubbed Framework", func(t *testing.T) {
+		mc := test.MustRunManagedCommand(t)
+		defer mc.Close()
```

Comment:
Out of curiosity, what is the reasoning behind the defer .Close() here?

Reply:
`test.MustRunManagedCommand()` starts up an serverless instance of FeatureBase. The deferred `Close()` just ensures that that running process is closed cleanly and releases any resources it's holding onto. It would probably get cleaned up either way, but this is good practice. One thing we add is a wrapper around the defer which checks the error on `Close()` and fails the test if it doesn't close cleanly.